src/plugins/initiatives/initiativesDirectory.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+import {type InitiativeFile, fromJSON, toJSON} from "./initiativesDirectory";
+
+const exampleInitiativeFile = (): InitiativeFile => ({
+  title: "Sample initiative",
+  timestampIso: ("2020-01-08T22:01:57.766Z": any),
```

Comment:
the `any` cast here is a smell that suggests we should have a typechecked module for working with ISO timestamps

Reply:
Yeah, as mentioned it shows the type is not meant to be used outside the one file :]
For the test file we make an any-cast exception to match.